.vib/kibana/cypress/cypress/integration/kibana_spec.js

```diff
@@ -6,7 +6,7 @@ before(() => {
   skipTheWelcomeScreen();
 });
 
-it('allows uploading a file', () => {
+it('allows uploading data', () => {
   cy.get('a[data-test-subj="uploadFile"]').click();
   cy.get('input#filePicker').selectFile(
```

Comment:
We usually use `[type="file"]`, which should be more resilient. I think you can also use it here.